dissect/target/plugins/apps/editor/editor.py

```diff
@@ -0,0 +1,23 @@
+from dissect.target.plugin import NamespacePlugin, export
+
+COMMON_EDITOR_FIELDS = [
+    ("datetime", "ts"),
+    ("string", "editor"),
+    ("path", "source"),
+]
+
+
+class EditorPlugin(NamespacePlugin):
+    """Editor plugin."""
+
+    __namespace__ = "editor"
+
+    @export
+    def extensions(self) -> None:
+        """Yields installed extensions."""
+        raise NotImplementedError
+
+    @export
+    def history(self) -> None:
+        """Yields history of files."""
+        raise NotImplementedError
```

Comment:
Any reason to include these here? This does not seem to consistent with most of the other 'base' namespaced plugins, such as with the remote access and locate base plugins.

Reply:
Check. Would you mind adding the alias? Then ill approve this PR